### PR TITLE
Resolved issue #316 Podcast cannot be saved in admin interface

### PR DESCRIPTION
**Pull request by fundatillus**
Corrected various "blank=False" or missing "blank=" instances to allow saving of Podcasts in Admin Interface.

### CODE REVIEW

**Comment by stefankoegl** [comment]
Looks good! I noticed some issues though:
* the migrations also change episode and user profile models, but those changes are missing from the PR
* the tests fail (probably due to incorrect code formatting; fix with “make format-code”)

Will check details when I’m not on mobile.

File: mygpo/users/migrations/0016_auto_20191230_2319.py [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+    dependencies = [
+        ('users', '0015_case_insensitive_username'),
+    ]
+
+    operations = [
+        migrations.AlterField(
+            model_name='userprofile',
+            name='twitter',
+            field=models.CharField(blank=True, max_length=15, null=True),
+        ),
```
Comment (stefankoegl) [inline mygpo/users/migrations/0016_auto_20191230_2319.py]: You have included a migration for a change in the `UserProfile.twitter` field, but I do not see a corresponding change in the model file. Those two (model and migratons) always need to be kept in sync.

Comment (fundatillus) [reply mygpo/users/migrations/0016_auto_20191230_2319.py]: Deleted the migration since it did not make any changes to the model.

**Comment by stefankoegl** [comment]
Thanks! I've just deployed the PR, and have started documentation on the deployment steps.

**Pull request merged**